media-gfx/imagemagick/imagemagick7-7.0.7.22.recipe

```diff
@@ -63,14 +70,32 @@ PROVIDES_devel="
 	cmd:MagickCore_config$secondaryArchSuffix
 	cmd:MagickWand_config$secondaryArchSuffix
 	cmd:Wand_config$secondaryArchSuffix
-	devel:libMagick++_7.Q16HDRI$secondaryArchSuffix = 7.0.0 compat >= 7
-	devel:libMagickCore_7.Q16HDRI$secondaryArchSuffix = 4.0.0 compat >= 4
-	devel:libMagickWand_7.Q16HDRI$secondaryArchSuffix = 4.0.0 compat >= 4
+	devel:libMagick++_7.Q16HDRI$secondaryArchSuffix = $libMagickVersionCompat
+	devel:libMagickCore_7.Q16HDRI$secondaryArchSuffix = $libMagickCoreVersionCompat
+	devel:libMagickWand_7.Q16HDRI$secondaryArchSuffix = $libMagickWandVersionCompat
 	"
 REQUIRES_devel="
 	imagemagick7$secondaryArchSuffix == $portVersion base
 	"
 
+defineDebugInfoPackage imagemagick7$secondaryArchSuffix \
+	$binDir/animate \
+	$binDir/compare \
+	$binDir/composite \
+	$binDir/conjure \
+	$binDir/convert \
+	$binDir/display \
+	$binDir/identify \
+	$binDir/import \
+	$binDir/magick \
+	$binDir/magick-script \
+	$binDir/mogrify \
+	$binDir/montage \
+	$binDir/stream \
+	$libDir/libMagick++-7.Q16HDRI.so.${libMagickVersion%%.*} \
+	$libDir/libMagickCore-7.Q16HDRI.so.${libMagickCoreVersion%%.*} \
+	$libDir/libMagickWand-7.Q16HDRI.so.${libMagickWandVersion%%.*}
```

Comment:
I'd recommend this variant:
```diff
-	$libDir/libMagick++-7.Q16HDRI.so.${libMagickVersion%%.*} \
+	$libDir/libMagick++-7.Q16HDRI.so.$libMagickVersion \
-	$libDir/libMagickCore-7.Q16HDRI.so.${libMagickCoreVersion%%.*} \
+	$libDir/libMagickCore-7.Q16HDRI.so.$libMagickCoreVersion \
-	$libDir/libMagickWand-7.Q16HDRI.so.${libMagickWandVersion%%.*}
+	$libDir/libMagickWand-7.Q16HDRI.so.$libMagickWandVersion
```
(This has the advantage of producing an error if we forget to update any of the `lib*Version` variables.)

Reply:
The lib versions for the debuginfopackage rely on the previous, it didn't work with "4.0.0" it needed "4" ...

Reply:
Well, I tested both of my builds (x86 secondary arch and x86_64) with the above suggestion, and everything was OK.

Reply:
Let me check to be sure ...

Reply:
You were right, updated ...